polling_stations/templates/home.html

```diff
@@ -70,3 +70,7 @@ <h2>{% blocktrans with election_date=election_date|date:"DATE_FORMAT" context="S
         </div>
     {% endblock page_content %}
 {% endblock content %}
+{% block in_page_javascript %}
+    <script data-goatcounter="https://wheredoivote.goatcounter.com/count"
+        async src="//gc.zgo.at/count.js"></script>
+{% endblock in_page_javascript %}
```

Comment:
Mostly want thumbs up that the `in_page_javascript` tag is an appropriate place for this.